File: Assigment/Module 2/2. Conditional logical Program/task1.c
// Program to check if two integers are equal

#include <stdio.h>

main()
{
    
    int num1, num2;

    printf("Enter the first integer: ");
    scanf("%d", &num1);
 
    printf("Enter the second integer: ");
    scanf("%d", &num2);

    if (num1 == num2) 
    {
        // If they are equal, print a message indicating so
        printf("The two integers are equal.\n");
    } 
    else 
    {
        // If they are not equal, print a message indicating so
        printf("The two integers are not equal.\n");
    }

    
}

File: Assigment/python fundamental/task1.py
# Program to find if a number is greater than or less than another number

# Input two numbers
num1 = float(input("Enter the first number: "))
num2 = float(input("Enter the second number: "))

# Compare the numbers using if-else
if num1 > num2:
    print(f"{num1} is greater than {num2}")
elif num1 < num2:
    print(f"{num1} is less than {num2}")
else:
    print(f"{num1} is equal to {num2}")